### DISCUSSION TITLE
Set Storage and Transfer from the command line and the port?

### DISCUSSION THREAD

**Discussion by black-jmyntrn** [outdoorbits/little-backup-box#250] [General]
for the life of me, i can't find the file to edit so I can change the port and actual storage and transfer drives. am i missing that in some instructions out there?

**Comment by outdoorbits** [comment]
What are you looking for? Lbb ist managed and configured by it's web UI.

**Comment by black-jmyntrn** [comment]
really to change the port to something other than 8000

**Reply by outdoorbits** [reply]
What's about using Https and port 443?

**Reply by black-jmyntrn** [reply]
The problem is I want to run Bitfocus Companion at the same time and it needs to use both 8000 and 443, I want to change LBB ports. Is that not possible?

**Comment by outdoorbits** [comment]
You can edit apache_ports.py and replace 8000 by whatever you want. This works of you use Comitup. Else just edit /etc/apache2/ports.conf.
Then you have to edit /etc/apache2/sites-available/little-backup-box.conf and do the same.
... and reboot.

**Reply by black-jmyntrn** [reply]
oh my wow! thank you! I did find the apache_ports file and made changes a few days ago but couldn't figure out why they didn't stick!

**Comment by black-jmyntrn** [comment]
what about setting the target and source permanetly? I am using the Siig JU-SA0S12-S1 USB 3.0 to SATA Hard Drive with SD Reader with a SATA NVME drive as my target but for some reason, every time I insert a SD card, it wants to sync to the SD card and the TARGET drive gets unset... even after update with the new drop downs under modifications. 

I also have a stream deck on this LLB running companion through the VENTION USB Hub

**Comment by outdoorbits** [comment]
You are running a usb to usb backup, right? Do you connect both drivers before starting the backup? If yes, please try the following:

  - Disconnect your source, target can stay connected
  - start backup
  - check your display or the log monitor and wait until target is mounted and until you are asked to connect source

Does this solve the problem?

**Comment by black-jmyntrn** [comment]
unfortunately, it does not and I always start it with the source only then insert the target.

**Comment by outdoorbits** [comment]
Could you please send your log file? Please set log to maximum.
